service/kvmconfig/cluster_framework.go

```diff
@@ -25,32 +25,19 @@ import (
 	"github.com/giantswarm/kvm-operator/service/kvmconfig/v9"
 )
 
-type FrameworkConfig struct {
+type ClusterFrameworkConfig struct {
```

Comment:
This is aligned with the aws-operator. There is a cluster and a drainer framework now.